bucket/matterbridge.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "1.22.0",
+    "description": "Bridge between mattermost, IRC, gitter, xmpp, slack, discord, telegram, rocketchat, twitch, ssh-chat, zulip, whatsapp, keybase, matrix, microsoft teams, nextcloud, mumble, vk and more with REST API.",
```

Comment:
- Nonsense to list random list of X chat applications
- Do not explain what this application do

Reply:
Not sure what you want me to write there, the application bridges or creates a bridge betwen those list of X chat applications.